README.md

```diff
@@ -248,6 +248,6 @@ From the start Bagging was built knowing that not all people speak English. If y
 There is a very active community around digital archiving. One of which is The Digital Curation Google Group (https://groups.google.com/d/forum/digital-curation) which  is an open discussion list that reaches many of the contributors to and users of this project.
 
 ## Roadmap for this library
-* Maintain interoperability with the [BagIt specification](https://tools.ietf.org/html/draft-kunze-bagit)
-* Fix bugs/issues reported (on going)
-* Translate to various languages (on going)
+* Maintain interoperability with the [BagIt specification](https://tools.ietf.org/html/draft-kunze-bagit).
```

Comment:
We should update the link with the current published version instead of the draft. Use https://tools.ietf.org/html/rfc8493